design_docs/openapi.yml

```diff
@@ -146,50 +146,44 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorResponse'
+            $ref: "#/components/schemas/ErrorResponse"
     NotFound:
       description: Not Found
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorResponse'
+            $ref: "#/components/schemas/ErrorResponse"
     InternalServerError:
-      description: Internal Server Error content:
+      description: Internal Server Error
+      content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorResponse'
+            $ref: "#/components/schemas/ErrorResponse"
   schemas:
     Conversation:
       type: object
       required:
+        - id
         - messages
-        - user_ip
-        - createdAt
-        - updatedAt
+        - timeCreated
       properties:
-        _id:
+        id:
           type: string
           description: The unique identifier for a conversation.
         messages:
           type: array
           items:
             $ref: "#/components/schemas/Message"
-        user_ip:
-          type: string
-          format: ipv4
-          description: The IP address of the user performing the conversation.
-        createdAt:
-          type: integer
-          description: The date the conversation was created.
-        updatedAt:
+        timeCreated:
```

Comment:
Why change the spec schema?

Reply:
b/c right now we're storing `timeCreated` in DB so i thought it'd be easier to keep API and DB consistent on this.

Reply:
Why are we using `timeCreated` in the db? The spec schemas for those also use `createdAt` and `updatedAt`.

Reply:
Ultimately it doesn't matter that much but we should try to stick to the specs IMO

Reply:
if there was a reason why, i don't remember. will update everything to `(created|updated)At`